lib/notification/mailgun/wrapper.js

```diff
@@ -0,0 +1,27 @@
+import HttpStatus from 'http-status-codes';
```

Comment:
This should be in the following folders
 lib/mongoose/js/src/config/email/mailgun.js
 lib/mongoose/ts/src/config/email/mailgun.ts
 lib/sequelize/js/src/config/email/mailgun.js
 lib/sequelize/ts/src/config/email/mailgun.ts

It should also be named mailgun and not wrapper

Reply:
Same as this